iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/Scalable.java

```diff
@@ -227,24 +235,47 @@ static StackScalable stack(Injection<?>... injections) {
         return new StackScalable(injectionScalables);
     }
 
+    static StackScalable stack(double minValue, double maxValue, Injection<?>... injections) {
```

Comment:
minValue and maxValue parameters should be after the injections parameter in order to be consistent between all the methods

Reply:
Sadly this is not possible with java's variable argument list.

Reply:
that's sad indeed :(